_maps/configs/pgf_crying_sun.json

```diff
@@ -21,29 +21,30 @@
 			"officer": true,
 			"slots": 1
 		},
-		"Bridge Crew": {
-			"outfit": "/datum/outfit/job/gezena/assistant/bridge",
+		"Marine Sergeant": {
```

Comment:
Change this to Marine Lieutenant so it's actually an officer. There's enough confusion about this slot's place in the chain of command as it is.

Reply:
I'd argue against the LT, personally. I liked the idea of it being changed to "Squad Leader" or "Marine Leader". It's more flexible and at the end of the day, while the "Sergeant" may be lower rank, their authority comes from their specialization.

Reply:
The thing is theoretically plenty of the Navy jobs could just outrank the Sarge, in theory, since their actual rank isn't specified beyond not being an officer. Changing it to a vague "Marine Leader" has the same issue. If this job is the ship's second in command, it needs to explicitly be an officer rank.

Reply:
Please refer to the PGF page for difference in command between marine and navy staff.